src/library_webgl2.js

```diff
@@ -630,23 +646,23 @@ var LibraryWebGL2 = {
     assert((value & 3) == 0, 'Pointer to integer data passed to glClearBufferiv must be aligned to four bytes!');
 #endif
 
-    GLctx.clearBufferiv(buffer, drawbuffer, HEAP32, value>>2);
+    GLctx.clearBufferiv(buffer, drawbuffer, HEAP32, {{{ ptrToIdx('value', 2) }}});
```

Comment:
Agreed, since there is now a more standardized way to make this conversion i will use it (these index conversion macros are part of an old commit from @juj that i pulled in to get some of this to work which i did not want to change too much).